### PR TITLE
release: Update README.md

### PR DESCRIPTION
**Pull request by hershpa**
Optimize and clean up the readme content for clarity. Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Comment by hershpa** [comment]
@chaitanya1731, can you help review this PR?

File: release/README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+* **RepositoryName:Tag** image naming pattern
   * **RepositoryName:** intel-data-center-gpu-driver-container
   * **Tag:** DriverVersion--KernelVersion
     * **DriverVersion:** X.Y.Z
-      * **X:** update for driver architecture change or major features 
+      * **X:** update for driver architecture or major feature change
         
-        **Example:** 0: drivers for RHEL8.x  1: drivers for RHEL9.x
-      * **Y:** updates for driver bug fixing or minor features support
+        **Example:** 0 = drivers for RHEL8.x,  1 = drivers for RHEL9.x
```
Comment (uMartinXu) [inline release/README.md]: should we continue use : instead of = since : is used in many other places here?

Comment (hershpa) [reply release/README.md]: Updated. :)

**Comment by chaitanya1731** [comment]
@uMartinXu @hershpa  PR looks good to me.

**Comment by uMartinXu** [comment]
This PR looks good to me. Thanks!

**Pull request merged**